src/deluge/gui/views/view.cpp

```diff
@@ -1259,7 +1263,7 @@ void View::setModLedStates() {
 		if (playbackHandler.recording == RecordingMode::ARRANGEMENT) {
 			indicator_leds::blinkLed(IndicatorLED::SESSION_VIEW, 255, 1);
 		}
-		else if (getRootUI() == &arrangerView) {
+		else if ((getRootUI() == &arrangerView) || (getRootUI() == &automationView && automationView.onArrangerView)) {
```

Comment:
This is an example of why I don't believe it should be a rootUI - it would be much cleaner if automation view opened as a layer instead of using changeRootUI

Looking at the code it's not actually hard - just needs to open with openUI instead and some checks change to use the currentUI instead of the rootUI

Same applies to keyboard and performance